frontend/src/components/page_content/plan-content.js

```diff
@@ -20,6 +22,10 @@ class PlanContent extends React.Component {
     // submit form
   }
 
+  componentDidMount() {
```

Comment:
i dont think this is necessary because its reset already in the page template component